src/integration-tests/model-in-image/mii-sample-wrapper/generate-sample-doc.sh

```diff
@@ -132,6 +137,9 @@ for phase in initial update1 update2 update3 update4; do
   if [ $configmap != "None" ]; then
     export INCLUDE_MODEL_CONFIGMAP=true
   fi
+  if [ $corrected_datasource_secret = "true" ]; then
+    export CORRECTED_DATASOURCE_SECRET=true
```

Comment:
```
if [ $corrected_datasource_secret = "true" ]; then
    export CORRECTED_DATASOURCE_SECRET=true
fi
```
should be replaced with
```
export CORRECTED_DATASOURCE_SECRET=$corrected_datasource_secret
```

Reply:
yes, this is better, but what it was before would also work